src/actions/users.js

```diff
@@ -241,7 +241,9 @@ export function getProfiles(page = 0, perPage = General.PROFILE_CHUNK_SIZE) {
         let profiles;
         try {
             profiles = await Client4.getProfiles(page, perPage);
-            removeUserFromList(currentUserId, profiles);
+            if (skipCurrent) {
```

Comment:
why is this conditional?

The problem by not skipping the current user all the time is that you will replace the currentUser with whatever getProfiles returns and then cause of the profile being sanitize you'll remove the `notifiy_props` needed by the currentUser

Reply:
tested, dont need this change here, closing...